pulp_2to3_migration/app/plugin/content.py

```diff
@@ -321,6 +318,9 @@ def get_remote_by_importer_id(importer_id):
                                 remote=None,
                                 deferred_download=False)
                             d_artifacts.append(da)
+                    for lce in pulp2lazycatalog:
+                        lce.is_migrated = True
```

Comment:
Could you use  ```pulp2lazycatalog.update(is_migrated=True)``` instead?

Reply:
there is a bulk operation that updates this later https://github.com/pulp/pulp-2to3-migration/blob/2ba2100c811ce8312e67effd78ce77d01aef1b47/pulp_2to3_migration/app/plugin/content.py#L442

Reply:
plus, we need to update them to True only after RemoteAftifact stage

Reply:
Gotcha